{{ .ProjectSnake }}/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
this file doesn't belong - we don't have "example application code" for the other languages.

If we wanted to add such a thing, we'd use https://hay-kot.github.io/scaffold/templates/scaffold-file.html#template-scaffolds - that would be cool but should be another PR I think